waku/v2/node/wakunode2_rln.go

```diff
@@ -73,7 +74,12 @@ func (w *WakuNode) mountRlnRelay(ctx context.Context) error {
 		}
 	}
 
-	rlnRelay, err := rln.New(w.Relay(), groupManager, w.opts.rlnRelayPubsubTopic, w.opts.rlnRelayContentTopic, w.opts.rlnSpamHandler, w.timesource, w.log)
+	var limiter *rate.Limiter
+	if w.opts.rlnRelayBandwidthThreshold != 0 {
+		limiter = rate.NewLimiter(rate.Limit(w.opts.rlnRelayBandwidthThreshold), w.opts.rlnRelayBandwidthThreshold)
```

Comment:
Wondering if we should we permit a burst of full bandwidth limit.
Generally bursts are set to a percentage of full-rate especially when rate limits are high like this.

Reply:
hm. not sure, tho. I set the burst and rate limit to be the same to match nwaku's behavior (since nwaku's rate limiter does not have the concept of bursts)

Reply:
Interesting, generally rate-limiters define smaller bursts so that all of the limits are not consumed (especially when limits are high). This is to ensure system is not hit with too big of a burst that it can't handle or causes it to slow down, it is a way to throttle. 

Now that i am rethinking, probably in this case since we are using this rate-limiter only to determine whether to enable proof verification, keeping burst same as rate should be fine.